README.md

```diff
@@ -14,8 +14,8 @@ If you are an Outreachy Applicant, start with reading [this](https://github.com/
 Setup for developers
 --------------------
 
-1. Make sure you have installed Python 3.4 (preferably latest minor release),
-   [pip](https://pip.pypa.io/en/latest/) and [virtualenv](http://www.virtualenv.org/en/latest/).
+1. Make sure you have installed Python 3.4 or 3.5 or 3.6(preferably latest minor release),
```

Comment:
In Travis we have only enabled 3.6, so would be better to only suggest just 3.6, added a commit in this; think can enable Travis for 3.4 and 3.5, not sure may be few Django dependent packages versions needs to be changed.